microshift_welcome/index.adoc

```diff
@@ -1,25 +1,55 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="microshift-welcome-index"]
-= {product-title} {product-version} Documentation
+[id="microshift-understanding"]
+= Understanding {product-title}
 include::_attributes/attributes-microshift.adoc[]
-:context: microshift-welcome-index
+:context: microshift-understanding
+
+toc::[]
 
 [.lead]
+
 Welcome to the official {product-title} {product-version} documentation, where you can learn about {microshift-short} and start exploring its features.
 
+[id="microshift-product-documentation_{context}"]
+== {microshift-short} product documentation
+
 To browse the {microshift-short} {product-version} documentation, use one of the following methods:
 
 * Use the navigation bars and links to browse.
 * Select the task that interests you from the contents of this Welcome page.
 
 To get started with {microshift-short}, use the following links:
-
-* xref:../microshift_getting_started/microshift-understanding.adoc#microshift-understanding[Understanding {product-title}]
-* xref:../microshift_install/microshift-install-rpm.adoc#microshift-install-rpm[Installing {product-title}]
+//text is in main assembly for the sake of cross references
 * xref:../microshift_release_notes/microshift-4-17-release-notes.adoc#microshift-4-17-release-notes[{product-title} release notes]
+* xref:../microshift_install/microshift-install-rpm.adoc#microshift-install-rpm[Installing {product-title}]
```

Comment:
this list is rendering strange:
<img width="772" alt="Screenshot 2024-07-31 at 11 13 02 AM" src="https://github.com/user-attachments/assets/e9091e25-a3cc-416a-bc31-37c39615bcd4">
though nothing noticeable jumps out at me as to why it's doing that 😕 

some suggestions to try:
- insert a blank line between the first list item and the leading sentence:
```suggestion

* xref:../microshift_release_notes/microshift-4-17-release-notes.adoc#microshift-4-17-release-notes[{product-title} release notes]
* xref:../microshift_install/microshift-install-rpm.adoc#microshift-install-rpm[Installing {product-title}]
```

- [enclose the list in an open block](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#enclose-in-open-block):
```suggestion
--
* xref:../microshift_release_notes/microshift-4-17-release-notes.adoc#microshift-4-17-release-notes[{product-title} release notes]
* xref:../microshift_install/microshift-install-rpm.adoc#microshift-install-rpm[Installing {product-title}]
--
```

Reply:
Wonder if it's because I put a comment in there and asciidoc reads that as now no line space, so it broke the bullets.

Reply:
Yep fixed! TY!.